submissions/MarharytaBoichenko/dom_menu/js/index.js

```diff
@@ -0,0 +1,38 @@
+import { navList, container } from "./refs.js";
+import quotations from "../data/quotations.json" assert { type: "json" };
+
+const createNavigation = () => {
+  const buttons = [];
+  for (let i = 0; i < 10; i++) {
```

Comment:
What if suddenly data in the `quotations` file changed? Please create buttons based on the data from quotations. Functions like `.map` should help you.